seed/static/seed/Terms_of_Service.txt

```diff
@@ -0,0 +1,34 @@
+Terms of Service ("Terms")
```

Comment:
@nllong @axelstudios @adrian-lara please review this and edit/make suggestions.

Reply:
Did you grab this from another set of ToS?